airbyte-webapp/src/components/JobItem/components/AttemptDetails.tsx

```diff
@@ -52,9 +52,17 @@ const AttemptDetails: React.FC<IProps> = ({ attempt, className, configType }) =>
     })}: ${failureOrigin}`;
   };
 
+  const truncateExternalMessage = (failureMessage?: string) => {
```

Comment:
There's also a way to do this with CSS:
https://nikitahl.com/pure-css-truncate-text

Reply:
+1 on this method!